.github/workflows/lambda_production.yml

```diff
@@ -7,7 +7,6 @@ on:
     branches: [master]
 
 env:
-  GITHUB_SHA: ${{ github.sha }}
```

Comment:
GitHub sets this environment variable for us.

Reply:
oh good to know.. 👍